packages/progress-bar-react/src/ProgressBar.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+interface Props {
+    progress: { current: number; total: number };
+    className?: string;
+    progressRole?: string;
+}
+
+export const calculatePercentage = ({ current, total }: { current: number; total: number }) =>
+    total === 0 ? 0 : (current * 100) / total;
+
+/**
+ *  Jøkul ProgressBar component
+ * @param progress
+ * @param className
+ * @param progressRole if set screen readers will read this string, if not the percentage is read
+ */
+export const ProgressBar: React.FC<Props> = ({ progress, progressRole, className }) => {
+    const width = `${calculatePercentage(progress)}%`;
+
+    return (
+        <aside
+            className={`jkl-progress-bar ${className}`}
+            role="progressbar"
+            aria-valuenow={progress.current}
+            aria-valuemin={0}
+            aria-valuetext={progressRole}
+            aria-valuemax={progress.total}
+            data-testid="jkl-progress-bar"
+        >
+            <span className="jkl-progress-bar__tracker" style={{ width }} data-testid="jkl-progress-bar__tracker" />
+        </aside>
```

Comment:
Hvorfor ikke bare bruke `<progress>`? Det er jo automatisk accessible, og støttes tilbake til IE10. Brukte et kvarter på å skrive litt om, og det funker flott!:

```jsx
<progress className={`jkl-progress-bar ${className}`} value={progress.current} max={progress.total}>
    {progressRole || percentageText}
</progress>
```
og 
```scss
.jkl-progress-bar {
    appearance: none;
    background-color: transparent;
    border: 0;
    height: $height;
    width: 100%;
    overflow: hidden;
    color: $ux-gronn;

    &::-webkit-progress-bar {
        background-color: transparent;
    }
    &::-webkit-progress-value {
        background-color: $ux-gronn;
        transition-property: width;
        @include motion("easeInBounceOut", "lazy");
    }

    &::-moz-progress-bar {
        background-color: $ux-gronn;
    }
}
```
gir samme visuelle resultat, med unntak at man må trikse litt for å få animasjon i Firefox.

Reply:
Og hvis vi lar elementene i `progress`-objektet være optional, kan vi få indeterminate-state på kjøpet om man bare sender et tomt objekt 😊 kan stiles med `.jkl-progress-bar:not([value])` eller bare `.jkl-progress-bar:indeterminate`

Reply:
Bytter kompleksitet i html med kompleksitet i css, jeg har ikke sterke meninger ene eller andre veien.

Reply:
Mener bare det er riktig å bruke semantiske HTML-elementer der man kan. Men er ikke imot å ta det inn slik det er heller (glemte å approve da jeg kommenterte, så gjorde det nå), ser jo ut som en solid implementasjon.

Reply:
Ja, jeg er enig, vi bør foretrekke bruk av semantiske elementer. Men når implementasjonen og støtten er fragmentert, så er det litt vanskeligere å heie på "standarden". 

Problemet her er animasjon i firefox. De har bare et pseudo element for å targete progressbaren, webkit har to. Dermed støtter chrome animasjon helt fint (gitt at man bruker -webkit prefix ( som jeg ikke liker at vi setter eksplisitt)), men firefox kan ikke animere. Fant en vei rundt, hvor vi endrer et custom property, og bruker den til å animere, men det er en del ekstra kode, som er unødvendig for de fleste brukerene våre, og er det litt hacky. Pluss at det ser litt derpy ut om man endrer verdien før den er ferdig å animere, så man må inn med throttling eller deboucing som igjen øker kode-fotavtrykket.